src/components/cardsPage.js

```diff
@@ -425,6 +425,7 @@ export const Wrapper = styled.section`
       flex-flow: row wrap;
       padding: 0 20px;
       margin: 10px 0 0;
+      max-width: 700px;
```

Comment:
Might be better to apply this to `.search-section` in case the h4 tag is super long?